src/ServiceControl/SagaAudit/SagaAuditing.cs

```diff
@@ -19,7 +19,7 @@ protected override void Setup(FeatureConfigurationContext context)
             context.Container.ConfigureComponent<SagaRelationshipsEnricher>(DependencyLifecycle.SingleInstance);
         }
 
-        class SagaRelationshipsEnricher : ImportEnricher
+        public class SagaRelationshipsEnricher : ImportEnricher
```

Comment:
It actually is `internal` because it is a nested class. Previously it was `private`.